blackbox/docs/sql/scalar.txt

```diff
@@ -1266,8 +1266,101 @@ It can be used to remove elements from array fields.
     +------------------------------+
     SELECT 1 row in set (... sec)
 
-Conditional Functions
-=====================
+Conditional Functions and Expressions
+=====================================
+
+case when ... then ... end
+--------------------------
+
+The ``case`` expression is a generic conditional expression similar to
+if/else statements in other programming languages and can be used
+wherever an expression is valid.
+
+::
+
+  CASE WHEN condition THEN result
+       [WHEN ...]
+       [ELSE result]
+  END
+
+Each *condition* expression must result in a boolean value. If the
+condition's result is true, the value of the *result* expression
+that follows the condition will be the final result of the ``case``
+expression. The remainder will not be proccessed. If the condition's
```

Comment:
... expression and the subsequent ``when`` branches (or conditions will not be processed.